implementations/rust/ockam/ockam_command/src/kafka/util.rs

```diff
@@ -104,10 +104,17 @@ pub async fn rpc(ctx: Context, (opts, args): (CommandGlobalOpts, ArgOpts)) -> mi
                     .to_string()
                     .color(OckamColor::PrimaryResource.color())
             ) + &fmt_log!(
-                "Brokers port range set to {}",
+                "Brokers port range set to {}\n\n",
                 &brokers_port_range
                     .to_string()
                     .color(OckamColor::PrimaryResource.color())
+            ) + &fmt_log!(
+                "{}\n",
+                "Kafka clients v3.4.x are supported.".color(OckamColor::FmtWARNBackground.color())
+            ) + &fmt_log!(
+                "{}\n",
+                "You can find the version you have with 'kafka-console-consumer.sh --version'."
```

Comment:
@nazmulidris I suspect this will display the same message for both `kafka-producer` and `kafka-consumer`.

I think we should display `kafka-console-consumer.sh --version` for the `kafka-consumer` command and `kafka-console-producer.sh --version` for the `kafka-producer` command.

Reply:
@mrinalwadhwa Yes, currently the same instruction is output for `ockam kafka-consumer` and `ockam kafka-producer`. 

The reason I chose to display the instruction to run `kafka-console-consumer.sh --version` is because both `kafka-console-producer.sh` and `kafka-console-consumer.sh` produce the same output. Here's a screenshot of running these commands using the Kafka CLI tools:

![image](https://github.com/build-trust/ockam/assets/2966499/526ce152-31bf-409c-80f9-a0baef08115e)

If you think that the UX will be better to make that distinction then I will be happy to make this change 👍🏽. PLMK 🙏🏽 .